.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,94 @@
+name: UnitTests
+
+on:
+    - pull_request
+    - push
+
+jobs:
+    tests:
+        name: "PHP ${{ matrix.php }} + Composer ${{ matrix.dependency }} + Symfony ${{ matrix.symfony }}"
+        runs-on: ubuntu-latest
+
+        strategy:
+            matrix:
+                include:
+                    # Lowest PHP with the lowest dependencies for every supported lowest major version of the Symfony
+                    -   php: 5.5
+                        dependency: lowest
+                        symfony: '^2.0'
```

Comment:
this should be not needed after `dependency: lowest`, right?

Reply:
Yes, I think so according to your message https://github.com/php-coveralls/php-coveralls/pull/325#discussion_r748887831